docs/registry.md

```diff
@@ -34,11 +34,12 @@ Proposals can be of different type and include (`proposal_metadata`) different d
 2. Proposal to update the set of successful-proposal receiver contract addresses.
 This proposal takes a parameter that has two constructors. The proposal can add to,
 or remove from the set of addresses.
-3. Transfer proposals, that can transfer XTZ or tokens as well as update the
```

Comment:
There is an explanation of the `proposal_metadata` including the transfer proposals in `docs/treasury.md` that needs to be updated too.